modules/jaeger-deploy-production-es.adoc

```diff
@@ -0,0 +1,143 @@
+////
+This PROCEDURE module included in the following assemblies:
+- rhbjaeger-deploying.adoc
+////
+
+[id="jaeger-deploy-production_{context}"]
+= Deploying the Jaeger production strategy
+
+The `production` deployment strategy is intended for production environments, where long term storage of trace data is important, as well as a more scalable and highly available architecture is required.
+
+.Prerequisites
+
+* The Elasticsearch Operator must be installed.
+* An Elasticsearch instance must be deployed.
+* The Jaeger Operator must be installed.
+* Review the instructions for how to customize the Jaeger installation.
+* An account with the `cluster-admin` role.
+
+.Procedure
+
+. Log in to the {product-title} web console as a user with the `cluster-admin` role.
+
+. Create a new project, for example `jaeger-system`.
+
+.. Navigate to *Home* -> *Projects*.
+
+.. Click *Create Project*.
+
+.. Enter `jaeger-system` in the *Name* field.
+
+.. Click *Create*.
+
+. Navigate to *Operators* -> *Installed Operators*.
+
+. If necessary, select `jaeger-system` from the Project menu.  You may have to wait a few moments for the Operators to be copied to the new project.
+
+. Click the Jaeger Operator. On the *Overview* tab, under *Provided APIs*, the Operator provides a single link.
+
+. Under *Jaeger* click *Create Instance*.
+
+. On the *Create Jaeger* page, replace the default `all-in-one` yaml text with your production YAML configuration, for example:
+
++
+.Example jaeger-production.yaml file with Elasticsearch
+[source,yaml]
+----
+apiVersion: jaegertracing.io/v1
+kind: Jaeger
+metadata:
+  name: jaeger-production
+  namespace:
+spec:
+  strategy: production
+  ingress:
+    security: oauth-proxy
+  storage:
+    type: elasticsearch
+    elasticsearch:
+      nodeCount: 1
```

Comment:
Same question as in the storage section - if this is supposed to be production example, shouldn't it have nodeCount 3?